vulnerabilities/pipelines/exploitdb.py

```diff
@@ -0,0 +1,95 @@
+import csv
+import io
+import logging
+
+import requests
+
+from vulnerabilities.models import Alias
+from vulnerabilities.models import Exploit
+from vulnerabilities.models import VulnerabilityReference
+from vulnerabilities.models import VulnerabilityRelatedReference
+from vulnerabilities.pipelines import VulnerableCodePipeline
+
+logger = logging.getLogger(__name__)
+
+
+class ExploitDBImproverPipeline(VulnerableCodePipeline):
+    """
+    ExploitDB Improver Pipeline: Fetch ExploitDB data, iterate over it to find the vulnerability with
+    the specified alias, and create or update the ref and ref-type accordingly.
+    """
+
+    exploit_data = None
+
+    license_expression = "GPL-2.0"
+
+    @classmethod
+    def steps(cls):
+        return (
+            cls.fetch_exploits,
+            cls.add_exploit,
+        )
+
+    def fetch_exploits(self):
+        exploit_db_url = (
+            "https://gitlab.com/exploit-database/exploitdb/-/raw/main/files_exploits.csv"
+        )
```

Comment:
Nice to have a INFO log here.

```suggestion
        exploit_db_url = (
            "https://gitlab.com/exploit-database/exploitdb/-/raw/main/files_exploits.csv"
        )
        self.log(f"Fetching {exploit_db_url}")
```

Reply:
Done